dll/ntdll/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 spec2def(ntdll.dll def/ntdll.spec ADD_IMPORTLIB)
 
 add_definitions(
+    -DNTDDI_VERSION=0x05020400
```

Comment:
Since you say it corresponds to NTDDI_WS03SP1, I take the opportunity to ask you whether you know if there has been other changes in these structures in SP2.

Reply:
You can see all changes here: http://terminus.rewolf.pl/terminus/structures/ntdll/_TEB_x86.html

Reply:
Nice ref!